pkgs/development/interpreters/npiet/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchurl
+, gd
+, giflib
+, groff
+, libpng
+, tk
+}:
+
+stdenv.mkDerivation rec {
+  pname = "npiet";
+  version = "1.3f";
+
+  src = fetchurl {
+    url = "https://www.bertnase.de/npiet/npiet-${version}.tar.gz";
+    sha256 = "sha256-Le2FYGKr1zWZ6F4edozmvGC6LbItx9aptidj3KBLhVo=";
+  };
+
+  buildInputs = [ gd giflib libpng ];
+
+  nativeBuildInputs = [ groff ];
+
+  prePatch = ''
```

Comment:
We usually do manual hackery as a `postPatch` (because any `patches` anyone adds are presumably patches _against_ the distributed source.

Reply:
Thank you, I committed your suggestion.